sqlalchemy_filters/filters.py

```diff
@@ -187,7 +187,7 @@ def get_named_models(filters):
     return models
 
 
-def apply_filters(query, filter_spec):
+def apply_filters(query, filter_spec, do_auto_join=True):
```

Comment:
> We can just call this argument `auto_join`, like the function. It still expresses what we want to do, is shorter and we reuse an existing name.

I don't think that would work:

* https://github.com/juliotrigo/sqlalchemy-filters/blob/e0a9ba026c40af70e1d81d919ee995df2165b8c5/sqlalchemy_filters/filters.py#L16
* https://github.com/juliotrigo/sqlalchemy-filters/blob/e0a9ba026c40af70e1d81d919ee995df2165b8c5/sqlalchemy_filters/filters.py#L190
* https://github.com/juliotrigo/sqlalchemy-filters/blob/e0a9ba026c40af70e1d81d919ee995df2165b8c5/sqlalchemy_filters/filters.py#L230

because auto_join function is imported to the local scope of the module

```
from .models import Field, auto_join, get_model_from_spec, get_default_model
...
def apply_filters(query, filter_spec, auto_join=True):
...
    if auto_join:
        query = auto_join(query, *filter_models)
```

simple test case results in `TypeError: 'bool' object is not callable` because local variable would mask out the actual function

Reply:
That's correct, I missed that. Let's leave it as it is.